pkg/cli/builder/builder.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"unsafe"
 
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
```

Comment:
nit: 
```suggestion
	"github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
```

Reply:
Go format keeps bringing this `v1` back when I go to remove it so I believe that it is here to stay unless I'm missing something.

Reply:
Ah, yep. I forgot that Go treats `v*` packages as versioned packages.